src/LaunchDarkly.Client/EventProcessor.cs

```diff
@@ -64,35 +62,24 @@ public void Flush()
 
         private void BulkSubmit(IEnumerable<Event> events)
         {
+            var uri = new Uri(_config.BaseUri + "api/events/bulk");
             try
             {
-                var url = new Uri(_configuration.BaseUri + "api/events/bulk");
-
-                var json = JsonConvert.SerializeObject(events.ToList());
-
-                var httpWebRequest = (HttpWebRequest)WebRequest.Create(url);
```

Comment:
Using HttpWebRequest is no longer considered the right way to do things. HttpClient is the way to go: http://www.diogonunes.com/blog/webclient-vs-httpclient-vs-httpwebrequest/